metadata.json

```diff
@@ -25,27 +45,34 @@
       ]
     },
     {
-      "operatingsystem": "RedHat",
```

Comment:
I sorted the operating systems alphabetically so I could compare this file to the one in ploperations-account.

Reply:
I like well sorted files better anyhow :)